doc/api.md

```diff
@@ -180,6 +180,7 @@ Some transformation plugins also allow the use of the following optional fields:
 -   **expression**: indicates that the value of the target attribute will not be the plain value or the measurement, but
     an expression based on a combination of the reported values. See the
     [Expression Language definition](#expression-language-support) for details
+-   **skip_value**: indicates the value to discard attribute of be sent ti CB when as a result of `expression`.
```

Comment:
skip_value (like entity_name) or skipValue (like explicitAttrs)? I would say that all the fields added recently follow the camelCase style, but I'm not fully sure.

Anyway, I'm afraid the mixture of naming styles we have in the API would be difficult to solve due to backward compatibility :(

Reply:
skip_value like entity_name, since is at the same level